doc/articles/guides/raspberry-pi/raspberry-pi-intro.md

```diff
@@ -113,7 +113,7 @@ Once the templates are installed, you can scroll back up and see the list of Uno
 
 Now we have the moving parts installed on our Pi, we can spin up a brand new Uno solution with the following command;
 
-`dotnet new UnoApp -o unoapp1 -ios=false -android=false -macos=false -wasm=false -skia-wpf=false -skia-tizen=false && cd unoapp1`
+`dotnet new unoapp -o HelloPi --preset=blank --platforms=gtk --platforms="linux-fb" && cd unoapp1`
```

Comment:
Since you renamed the project name from `unoapp1`, Don't you also need to change where this name is being referenced?